app/javascript/components/FiltersBox.js

```diff
@@ -84,39 +173,28 @@ export default function FiltersBox(props) {
         </span>
       </p>
       <ul>
-        {
-          // TODO: Abstract to use Filters component
-          // after passing through function for onClick interaction
-          // (SCP-2109)
-          props.facet.filters.map((d) => {
-            const id = `filter-${facetName}-${d.id}`;
-            return (
-              <li key={'li-' + id}>
-                <input
-                  type="checkbox"
-                  aria-label="Checkbox"
-                  onClick={handleFilterClick}
-                  name={id}
-                  id={id}
-                />
-                <label htmlFor={id}>{d.name}</label>
-              </li>
-            );
-          })
-        }
+        <Filters
+          facetType='string'
+          filters={matchingFilters}
+          onClick={updateSelections}
+        />
       </ul>
       {/*
       TODO: abstracting this and similar code block in
       FacetsAccordionBox into new component (SCP-2109)
        */}
       <div className='filters-box-footer'>
-        <span>Clear</span>
+        <ClearFilters
```

Comment:
rather than passing a show prop, would it be cleaner to just have this be?   
{ showClear && <ClearFilters ... }
Usually I expect show/hide params for cases where you want the component to still occupy layout space (and so it would be using display:hidden instead of display:none)

Reply:
Neat!  Done in 960e9b5.